src/applications/check-in/utils/pilotFeatures/index.js

```diff
@@ -0,0 +1,82 @@
+import wilkesBarre from './travelClaim/wilkesBarre.json';
+// import salem from './travelClaim/salem.json';
+
+const devStations = (withDev = true, withStage = true, withLocal = true) => {
```

Comment:
What is the use case for this? Can we treat dev stations like the other lists?

Reply:
I was thinking that it would be a nice to have a way to easily set a feature to on locally but not in dev/stage or some mix of the three. It's just a bit of up front flexibility. I think the other lists may or may not be handled in external files depending on the feature anyway.

Reply:
If you think it will get used then I'm fine with it. Should there be a unit test for it?

Reply:
yeah maybe you are right. this is probably gilding the lily a bit. Hard to unit test since it is a private function and would need to write a bunch of setup in the public interface to do it. ok, you've convinced me! I'll simplify it.